src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -3694,6 +3893,30 @@ type TicketType implements Node {
     zones: [String!]
 }
 
+scalar TimeZone
```

Comment:
I don't think we need that anymore, or do we?

Reply:
No, this can be removed.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/910717a739f12e99b3a445c338fd24a8e4b73637